documentation/docs/30-advanced/20-hooks.md

```diff
@@ -39,6 +39,8 @@ export async function handle({ event, resolve }) {
 
 If unimplemented, defaults to `({ event, resolve }) => resolve(event)`.
 
+If you need to exclude some code from running during the build or prerendering step, check that the app is not [`building`]($app-environment#building) beforehand.
```

Comment:
Not sure if this suffices - the confusion is also about why `handle` even is executed

Reply:
Not quite - the question is also "why is that route even requested during prerendering, I did `export const prerender = false`"

Reply:
How about
```suggestion
During prerendering, SvelteKit crawls your pages for links and invokes the `handle` hook for each route it finds. If you need to exclude some code from running during this phase, check that the app is not [`building`]($app-environment#building) beforehand.
```